core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -899,13 +905,15 @@ public static VirtualChannel getChannelToMaster() {
     /**
      * Helper method for Jelly.
      */
+    @Restricted(DoNotUse.class)
+    @RestrictedSince("TODO")
```

Comment:
Good idea, but I'm calling it out in this comment anyway for more visibility among all the cleanup changes.

Reply:
agreed